src/paradigma/tremor/tremor_analysis.py

```diff
@@ -101,4 +106,99 @@ def detect_tremor_io(input_path: Union[str, Path], output_path: Union[str, Path]
+
+    # determine valid days
+    daytime_hours = range(config.daytime_hours_lower_bound,config.daytime_hours_upper_bound)
+    nr_windows_per_day = Counter([dt.date() for dt in time if dt.hour in daytime_hours])
+    valid_days = [day for day, count in nr_windows_per_day.items() 
+                  if count * config.window_length_s / 3600 >= config.valid_day_threshold_hr]
+    df_aggregates = pd.DataFrame({'nr_valid_days': [len(valid_days)]}) # save number of valid days
+
+    # remove windows during non-valid days, non-daytime hours and non-tremor arm movement
+    df_filtered = df.loc[[dt.date() in valid_days for dt in time]]
+    df_filtered = df_filtered.loc[[dt.hour in daytime_hours for dt in time]]
+    df_aggregates['nr_windows_daytime'] = df_filtered.shape[0] # save number of windows during daytime on valid days
+    df_filtered = df_filtered.loc[df_filtered.low_freq_power <= config.movement_threshold]
+    df_aggregates['nr_windows_daytime_rest'] = df_filtered.shape[0] # save number of windows during daytime on valid days without non-tremor arm movement
+
+    # calculate weekly tremor time
+    df_aggregates['tremor_time'] = np.sum(df_filtered['pred_tremor_checked']) / df_filtered.shape[0] * 100
+
+    # calculate weekly tremor power measures
+    tremor_power = df_filtered['tremor_power'][df_filtered['pred_tremor_checked']==1]
+    tremor_power_median, tremor_power_90th_perc, tremor_power_mode = aggregate_tremor_power(tremor_power,config)
+    df_aggregates['tremor_power_median'] = tremor_power_median
+    df_aggregates['tremor_power_mode'] = tremor_power_mode
+    df_aggregates['tremor_power_90th_perc'] = tremor_power_90th_perc
+
+    return df_aggregates
+
+
+def quantify_tremor_io(path_to_feature_input: Union[str, Path], path_to_prediction_input: Union[str, Path], output_path: Union[str, Path], config: TremorQuantificationConfig) -> None:
+    
+    # Load the features & predictions
+    metadata_time, metadata_samples = read_metadata(path_to_feature_input, config.meta_filename, config.time_filename, config.values_filename)
+    df_features = tsdf.load_dataframe_from_binaries([metadata_time, metadata_samples], tsdf.constants.ConcatenationType.columns)
+
+    metadata_dict = tsdf.load_metadata_from_path(os.path.join(path_to_prediction_input, config.meta_filename))
+    metadata_time = metadata_dict[config.time_filename]
+    metadata_samples = metadata_dict[config.values_filename]
+    df_predictions = tsdf.load_dataframe_from_binaries([metadata_time, metadata_samples], tsdf.constants.ConcatenationType.columns)
+
+    # Validate
+    # Dataframes have same length
+    assert df_features.shape[0] == df_predictions.shape[0]
+
+    # Dataframes have same time column
+    assert df_features[DataColumns.TIME].equals(df_predictions[DataColumns.TIME])
+
+    # Subset features
+    l_feature_cols = ['tremor_power','low_freq_power']
+    df_features = df_features[l_feature_cols]
```

Comment:
Tenzij je `l_feature_cols` nog een keer gebruikt zou ik hier één regel van maken

```suggestion
    df_features = df_features[['tremor_power', 'low_freq_power']]
```